src/app/home/work-item-widget/work-item-widget.component.html

```diff
@@ -3,13 +3,13 @@
     <div class="card-pf-heading-bg">
       <div class="card-pf-heading f8-card-heading">
         <h2 class="card-pf-title">My Work Items
-        <span *ngIf="recentSpaces.length !== 0">
+        <span *ngIf="spaces.length !== 0">
```

Comment:
Is there a specific issue about using recent spaces? Issues #3621 and #3504 don't appear to be related.

Using recent spaces was intentional here to limit the number of potential options shown in the drop down menu.

@mindreeper2420 @catrobson

Reply:
I'm fine with whatever folks decide to do here, just want to confirm with UXD that this is what we want for the UI. 

The other widgets for this dashboard (i.e., "recent spaces" and "Recent Active Pipelines") are showing recent spaces, so this could also be addressed with a label change.